CHANGELOG.md

```diff
@@ -3,6 +3,63 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.17.0
+
+_Oct 26, 2023_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- The Tree View package is now stable!
```

Comment:
```suggestion
- The Tree View package is now stable!
- The line charts support series with different domains
```

Since it's one of the most asked recently